common/src/main/java/com/skytouch/model/Product.java

```diff
@@ -0,0 +1,94 @@
+package com.skytouch.model;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+
+@JsonIdentityInfo(generator = ObjectIdGenerators.IntSequenceGenerator.class, property = "@id", scope = Product.class)
+public class Product implements Serializable {
+
+
+    private long id;
+
+    private String name;
+
+    private String description;
+
+    private double unitPrice;
```

Comment:
Why did you chose double as type for handling prices?
Execute this line please:
```
System.out.println(2.00D - 1.10D);
```
The expected result was _.9_, however it gave another number. why did it happen? and how is it related to Double and Long types?

Reply:
Sorry my mistake, this happens because when a variable "x" is converted to double there is a percentage of error (e0 or e1), to solve this problem i will change it to BigDecimal